libs/core-components/src/lib/Button/Button.tsx

```diff
@@ -12,7 +10,6 @@ export const TYPE_MAP = {
   active: 'is-active',
   link: 'as-link',
 };
-export const TYPES = [''].concat(Object.keys(TYPE_MAP));
```

Comment:
There is also a [`SIZES` export](https://github.com/TACC/tup-ui/pull/7/files#diff-5abb4fe35237cb8ae8ea69b06bc13bc9614458d6e34c2d2f92be5893a3b845aeR20). It can be deleted too, correct?

```
export const SIZES = [''].concat(Object.keys(SIZE_MAP));
```

Reply:
The unit tests use the SIZES export. We should consider redesigning the unit tests to be more explicit rather than dynamic to avoid exports like this.